### PR TITLE
Add user manual for Performance Collector with both English and Chinese version

### PR DESCRIPTION
**Pull request by songtao98**
This PR create user manual for Performance Collector with both English and Chinese version. 

Signed-off-by: songtao98 <songtao2603060@gmail.com>

### CODE REVIEW

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+- Koordinator 开启Performance Collector，节点资源概览：
+
+![image.png](https://intranetproxy.alipay.com/skylark/lark/0/2022/png/69756458/1669796897295-c395e829-b6d7-45db-8352-cf74485dd410.png#clientId=ua4f9b36f-9dcd-4&crop=0&crop=0&crop=1&crop=1&from=paste&height=91&id=ued1662d8&margin=%5Bobject%20Object%5D&name=image.png&originHeight=182&originWidth=2504&originalType=binary&ratio=1&rotation=0&showTitle=false&size=47531&status=done&style=none&taskId=u45218f76-5188-4452-b6c2-7b7ae1487ac&title=&width=1252)
+![image.png](https://intranetproxy.alipay.com/skylark/lark/0/2022/png/69756458/1669796918048-25f10081-5e2f-470d-9f9f-d399febaff4d.png#clientId=ua4f9b36f-9dcd-4&crop=0&crop=0&crop=1&crop=1&from=paste&height=684&id=u7c21303c&margin=%5Bobject%20Object%5D&name=image.png&originHeight=1368&originWidth=3672&originalType=binary&ratio=1&rotation=0&showTitle=false&size=733472&status=done&style=none&taskId=uf9afff7c-2c4a-49a7-b4be-bf2c089c4b5&title=&width=1836)
+
+- 无明显开销增长：
+   - System Mode CPU usage: 0.35% -> 0.49%
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:77]: 资源消耗需要控制在常见的资源水位上

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+   - perf_event_open(2)的使用
+   - PSI功能的开启
+## 实例
+
+1. 打开想要使用的Performance Collector：
+```shell
+helm install koordinator https://... --set featureGates="CPICollector=true\,PSICollector=true"
+```
+
+2. 使用如下flag配置指标采集器的时间窗口、采集间隔等：
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:89]: 可以用表格来描述更细节的相关args

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        - args:
+          	# Collect cpi interval by seconds
+            - '-cpi-collector-interval-seconds=60'
+      			# Collect cpi time window by seconds
+            - '-collect-cpi-timewindow-seconds=10'
+						# Collect psi interval by seconds
+            - '-psi-collector-interval-seconds=10'
+```
+
+3. 您可以在Prometheus指标暴露端口处观察到采集到的指标：
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:102]: chart里记得补充koordlet的service monitor

File: docs/user-manuals/performance-collector.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
+## Motivation
+In real production environment, the runtime state of a node is a "chaotic system", and application interference caused by resource contention cannot be absolutely avoided. Koordinator is building interference detection and optimization capabilities. By extracting metrics of application running status, real-time analysis and detection are carried out, and more targeted strategies are adopted for target applications and interference sources after interference is discovered. 
+Koordinator implements a series of `Performance Collectors` to collect low-level metrics highly correlated with application running status on one node, and expose them through `Prometheus` to provide support for interference detection capabilities and cluster application scheduling.
```
Comment (saintube) [inline docs/user-manuals/performance-collector.md:3]: 注意下 markdown 规范，标题前后空一行

File: docs/user-manuals/performance-collector.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+### Configuration
+Performance Collector is _Disabled_ currently by default. You can use it by setting feature-gates:
+```shell
+helm install koordinator https://... --set featureGates="PerformanceCollector=true\,PSICollector=true"
+```
+
+> Notice that at Koordinator version v1.0.0, the feature-gate PerformanceCollector actually enables capability of CPI collection only. We already marked this feature gate as deprecated, and plan to use CPICollector instead in future release.
+
+```shell
+helm install koordinator https://... --set featureGates="CPICollector=true\,PSICollector=true"
```
Comment (saintube) [inline docs/user-manuals/performance-collector.md:27]: typo `"CPICollector=true,PSICollector=true"`

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
+## 背景 
+在真实的生产环境下，单机的运行时状态是一个“混沌系统”，资源竞争产生的应用干扰无法绝对避免。Koordinator正在建立干扰检测与优化的能力，通过提取应用运行状态的指标，进行实时的分析和检测，在发现干扰后对目标应用和干扰源采取更具针对性的策略。
+Koordinator已经实现了一系列`Performance Collector`，在单机侧采集与应用运行状态高相关性的底层指标，并通过`Prometheus`暴露出来，为干扰检测能力和集群应用调度提供支持。
+## 使用方法
+### 准备条件
+
+- Kubernetes >= 1.18
+- Koordinator >= 1.0
+- 若您使用CPI Collector，请确保您的机器支持获取Cycles、Instructions PMU。
```
Comment (saintube) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:9]: PMU 加下全称吧，比如 PMU (Performance Monitoring Unit)

Comment (zwzhang0107) [reply i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:9]: 给一个检查是否支持的命令

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+   |            flag名称             | 默认值 |              含义               |
+   | :-----------------------------: | :----: | :-----------------------------: |
+   | -cpi-collector-interval-seconds |   60   | CPI指标采集的时间间隔，单位为秒 |
+   | -collect-cpi-timewindow-seconds |   10   | CPI指标采集的时间窗口，单位为秒 |
+   | -psi-collector-interval-seconds |   10   | PSI指标采集的时间间隔，单位为秒 |
+3. 您可以在Prometheus指标暴露端口处观察到采集到的指标：
+```shell
+# HELP koordlet_container_cpi Container cpi collected by koordlet
+# TYPE koordlet_container_cpi gauge
+koordlet_container_cpi{cid="", node="", pid=""} 1.447460161805656
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:144]: more detail about prometheus

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+
+2. 使用如下flag配置指标采集器的时间窗口、采集间隔等：
+
+   |            flag名称             | 默认值 |              含义               |
+   | :-----------------------------: | :----: | :-----------------------------: |
+   | -cpi-collector-interval-seconds |   60   | CPI指标采集的时间间隔，单位为秒 |
+   | -collect-cpi-timewindow-seconds |   10   | CPI指标采集的时间窗口，单位为秒 |
+   | -psi-collector-interval-seconds |   10   | PSI指标采集的时间间隔，单位为秒 |
+
+3. 您可以在Prometheus指标暴露端口（默认为9316）处观察到采集到的指标，查询 API为`/metrics`，CPI指标以*cycles*和*instructions*两条记录分开展示：
```
Comment (zwzhang0107) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/user-manuals/performance-collector.md:141]: 记录一个issue，helm chart中增加koordlet的service monitor监控

**Review by jasonliu747 — APPROVED** [review]

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**